src/js/experimental/menu/media-audio-track-menu.js

```diff
@@ -0,0 +1,136 @@
+import './media-chrome-menu-item.js';
+import { globalThis } from '../../utils/server-safe-globals.js';
+import { MediaUIAttributes, MediaUIEvents } from '../../constants.js';
+import { parseAudioTrackList } from '../../utils/utils.js';
+import {
+  MediaChromeMenu,
+  createMenuItem,
+  createIndicator,
+} from './media-chrome-menu.js';
+import {
+  getStringAttr,
+  setStringAttr,
+  getMediaController,
+} from '../../utils/element-utils.js';
+
+/**
+ * @attr {string} mediaaudiotrackenabled - (read-only) Set to the enabled audio track.
+ * @attr {string} mediaaudiotracklist - (read-only) Set to the audio track list.
+ */
+class MediaAudioTrackMenu extends MediaChromeMenu {
+  static get observedAttributes() {
+    return [
+      ...super.observedAttributes,
+      MediaUIAttributes.MEDIA_AUDIO_TRACK_LIST,
+      MediaUIAttributes.MEDIA_AUDIO_TRACK_ENABLED,
+      MediaUIAttributes.MEDIA_AUDIO_TRACK_UNAVAILABLE,
+    ];
+  }
+
+  #audioTrackList = [];
+  #prevState;
+
+  attributeChangedCallback(attrName, oldValue, newValue) {
+    super.attributeChangedCallback(attrName, oldValue, newValue);
+
+    if (
+      attrName === MediaUIAttributes.MEDIA_AUDIO_TRACK_ENABLED &&
+      oldValue !== newValue
```

Comment:
Will this ever be true? Feels odd if it could be.

Reply:
yes, the first condition is the attribute name.
the value is the audio track ID